chore: rename localrep to api

sergebouchut2 opened the pull request.

sergebouchut2 commented:
/e2e

Owlfred commented:
End to end tests: :hourglass_flowing_sand: **STARTING**

sergebouchut2 commented:
/e2e

Owlfred commented:
End to end tests: :hourglass_flowing_sand: **STARTING**

sergebouchut2 commented:
/e2e --mode distributed

Owlfred commented:
End to end tests: :hourglass_flowing_sand: **STARTING**

sergebouchut2 commented:
@AlexandrePicosson The failing test `test_compute_plan_transient_outputs` may be related to this [PR](https://github.com/Substra/substra-tests/pull/201)
```
 _____________________ test_compute_plan_transient_outputs ______________________
[gw2] linux -- Python 3.8.14 /usr/local/bin/python

factory = <substratest.factory.AssetsFactory object at 0x7f53ccd69b20>
client = <substratest.client.Client object at 0x7f53ccded070>
default_dataset = <substratest.factory.AugmentedDataset object at 0x7f53ccd63d60>

    @pytest.mark.slow
    @pytest.mark.remote_only
    def test_compute_plan_transient_outputs(factory: AssetsFactory, client: Client, default_dataset):
        """
        Create a simple compute plan with tasks using transient inputs, check if the flag is set
        """
        data_sample_1_input, data_sample_2_input, _, _ = default_dataset.train_data_sample_inputs
    
        # Register the Algo
        simple_algo_spec = factory.create_algo(AlgoCategory.simple)
        simple_algo = client.add_algo(simple_algo_spec)
    
        cp_spec = factory.create_compute_plan()
        traintuple_spec_1 = cp_spec.create_traintuple(
            algo=simple_algo,
            inputs=default_dataset.opener_input + [data_sample_1_input],
            outputs=FLTaskOutputGenerator.traintuple(transient=True),
        )
    
        cp_spec.create_traintuple(
            algo=simple_algo,
            inputs=default_dataset.opener_input
            + [data_sample_2_input]
            + FLTaskInputGenerator.trains_to_train([traintuple_spec_1.traintuple_id]),
        )
    
        cp_added = client.add_compute_plan(cp_spec)
        client.wait(cp_added)
    
        traintuple_1 = client.get_traintuple(traintuple_spec_1.traintuple_id)
        assert traintuple_1.outputs[OutputIdentifiers.model].is_transient is True
    
        # Validate that the transient model is properly deleted
        model = client.get_task_models(traintuple_spec_1.traintuple_id)[0]
>       client.wait_model_deletion(model.key)

tests/test_execution_compute_plan.py:794: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <substratest.client.Client object at 0x7f53ccded070>
model_key = '7196fc0c-0e58-44fc-9c32-0833a17779d0'

    def wait_model_deletion(self, model_key):
        """Wait for the model to be deleted (address unset)"""
        tstart = time.time()
        model = self._client.get_model(model_key)
        while model.address:
            if time.time() - tstart > self.future_timeout:
>               raise errors.FutureTimeoutError(f"Future timeout waiting on model deletion for ***model_key***")
E               substratest.errors.FutureTimeoutError: Future timeout waiting on model deletion for 7196fc0c-0e58-44fc-9c32-0833a17779d0

substratest/client.py:305: FutureTimeoutError
```

AlexandrePicosson commented:
Thank you for raising this issue, I'll check the logs as soon as I am available and keep you updated with a potential fix.

AlexandrePicosson commented:
Looks like the issue is due to an Orchestrator error, I'll try to submit a fix as soon as possible.

AlexandrePicosson commented:
It should be resolved with the merge of https://github.com/Substra/orchestrator/pull/15, sorry for the inconvenience.

AlexandrePicosson commented:
/e2e --mode distributed

Owlfred commented:
End to end tests: :hourglass_flowing_sand: **STARTING**

sergebouchut2 commented:
> It should be resolved with the merge of the following PR, sorry for the inconvenience.
> 
>     * [fix: rename DeleteOutput to DisableOutput in computetask smart contract orchestrator#15](https://github.com/Substra/orchestrator/pull/15)

No worry thanks for your reactivity!

cboillet-dev commented:
remaining occurences of `localrep`:
[`localrep_resync_events_page_size`](https://github.com/Substra/substra-backend/blob/chore/rename_localrep_to_api/backend/backend/settings/common.py#L424) django setting it could also be renamed

sergebouchut2 commented:
> remaining occurences of `localrep`: [`localrep_resync_events_page_size`](https://github.com/Substra/substra-backend/blob/chore/rename_localrep_to_api/backend/backend/settings/common.py#L424) django setting it could also be renamed

It seems not used anymore, introduced [here](https://github.com/Substra/substra-backend/commit/f6f225b502e940914098f54ece4a51046e66b6f8), removed [here](https://github.com/Substra/substra-backend/commit/4a5a65b3ed7c0c96e2a303ae9ba37a3a562493d9#diff-bbc5395dcf594e6092a31c207b688fe64be94a974d0b4fcc1a4190cfb1494be5)
Let's clean that in a separated PR
